plugin/src/_rules/position.js

```diff
@@ -1,4 +1,102 @@
 import { handler as h, insetMap } from '#utils';
+import { bounded } from "#bounding";
+
+
+const ORDER_BOUNDS = [1, 12];
+const numericHandler = { handler: (d) => h.number(d) };
+
+export const orders = [
+    [/^order-(\d+)$/,
+    bounded(
+      ([, d]) => ({ 'order': h.number(d)}),
+      ORDER_BOUNDS,
+      numericHandler
+    )],
+    ['order-first', { order: '-9999' }],
+    ['order-last', { order: '9999' }],
+    ['order-none', { order: '0' }],
+];
+
+export const justifies = [
+    // contents
+    ['justify-start', { 'justify-content': 'flex-start' }],
+    ['justify-end', { 'justify-content': 'flex-end' }],
+    ['justify-center', { 'justify-content': 'center' }],
+    ['justify-between', { 'justify-content': 'space-between' }],
+    ['justify-around', { 'justify-content': 'space-around' }],
+    ['justify-evenly', { 'justify-content': 'space-evenly' }],
+    // ...makeGlobalStaticRules('justify', 'justify-content'),
```

Comment:
These comments could either be nuked or with a TODO if there's still an action pending on them?

Reply:
We weren't sure how to approach this. Do you know who we should talk to if we need support for those global rules or if it makes sense to remove them?

Reply:
With all these rules that aren't in Tailwind, I think it's fair we start with not including it and see if the community comes up with a valid and broad usecase.

Especially since these aren't in Fabric, and haven't been mentioned/requested!

Reply:
Does it matter that much if they are in Tailwind or not anymore as long as they seam useful?(/and not silly duplicates with just alias names) , we are officially leaving the "our thing is the same as Tailwind" -train now anyways with a full docsite and everything:)

Reply:
Good point Snorre! Let's uncomment and keep them!